gprofiler/profilers/java.py

```diff
@@ -661,6 +750,15 @@ def read_output(self) -> Optional[str]:
             default=True,
             help="Disable collection of async-profiler meminfo at the end of each cycle (collected by default)",
         ),
+        ProfilerArgument(
+            "--java-jvm-flags-to-retrieve",
+            dest="java_jvm_flags_to_retrieve",
+            type=str,
+            nargs="?",
+            default=JavaFlagRetrievalOptions.DEFAULT.value,
+            help="Comma-separated list of JVM flags to retrieve from the JVM process, 'all' to retrieve all flags, "
+            "or 'default' to retrieve only non-default jvm flags. Defaults to '%(default)s'",
```

Comment:
Maybe "'non-default' to retrieve only 'non-default' JVM flags"? Makes more sense to me. Also fix `JavaFlagRetrievalOptions`.

Reply:
Okay, I read the code and I see that the "default" here means gProfiler's default, which goes beyond the non-default JVM flags (skips nonproduct flags, etc).
Partial documentation is the worst case - this misleads the reader. Either make it complete, or remove tiny bits and just write `or 'default' for the defaults filtering settings for flags; see filter_jvm_flag(). Defaults to ...`.